x/supply/types/querier.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+// query endpoints supported by the supply Querier
+const (
+	QueryTotalSupply = "totalSupply"
```

Comment:
```suggestion
	QueryTotalSupply = "total_supply"
```

Reply:
actually, we only use snake_case on distribution. The other modules use camelcase